hash/ibm5170.xml

```diff
@@ -12853,8 +12945,7 @@ license:CC0
 		<info name="version" value="1.000" />
 		<part name="flop1" interface="floppy_3_5">
 			<dataarea name="flop" size = "1474560">
-				<!-- has a saved configuration file -->
-				<rom name="Rise of the Dragon [Sierra] [1990] [3.5HD] [Disk 1 of 6].img" size="1474560" crc="feba710f" sha1="4692dda4267b4b4b0791c837e9bdb27a45e48240" status="baddump"/>
+				<rom name="Rise of the Dragon [Sierra] [1990] [3.5HD] [Disk 1 of 6].img" size="1474560" crc="434e737c" sha1="bccd718c8c2a3a26c1f8624e7f85b7eec7648b91"/>
```

Comment:
This looks the same as the old dump but the config file was deleted, which doesn't actually remove the contents of the file from the disk, it just marks it as "deleted" in the file allocation table. So unless it came like that from the factory this is still a bad dump.

Reply:
You're right! Keep the "bad dump" status flag, please.